test/unit_Interval.jl

```diff
@@ -93,6 +93,12 @@ for N in [Float64, Float32, Rational{Int}]
     E = intersection(A, Interval(N(0), N(1)))
     @test isempty(E)
 
+    # disjointness check
+    @test !isdisjoint(A, B)
+    @test isdisjoint(A, Interval(N(-1), N(1)), false)
+    s, w = isdisjoint(A, A, true)
+    @test s == false && w == [min(A)]
```

Comment:
Can we have a nontrivial test with a new interval `C` instead of `A` and `A`?
The concrete return value is an implementation detail and can change, so I would change the test to
```suggestion
    @test s == false && w ∈ A && w ∈ C
```